src/ui/osx/TogglDesktop/Feature/Timeline/Models/TimelineData.swift

```diff
@@ -14,6 +14,7 @@ final class TimelineData {
         case timeLabel = 0
         case timeEntry
         case activity
+        case background
```

Comment:
I have to convert the Background from Supplementary View (Footer) to Item View since Drop and Drop doesn't consider the Header, Footer and Decoration View as a drop target. As a result, it's unable to drop on it.